keda/Chart.yaml

```
@@ -4,7 +4,7 @@ description: Event-based autoscaler for workloads on Kubernetes
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.5.0
+version: 2.5.1
```

Let's keep the version as-is and take that through 2.6 since this is not exactly a fix
```suggestion
version: 2.5.0
```